### DISCUSSION TITLE
Show div id of content in modal after child closes

### DISCUSSION THREAD

**Discussion by mahmoud-m-abadi** [wire-elements/modal#285] [Q&A]
Hi there,

This a weird issue that I see it in this great package. When I close a child modal the parent modal shows like the attached image. Why?
![Screenshot from 2022-11-11 15-15-01](https://user-images.githubusercontent.com/9986497/201334201-ca1cea22-bd3a-47e6-95bf-e37943a53a24.png)

**Comment by PhiloNL** [comment]
I would recommend checking for unclosed `<divs>`

**Reply by mahmoud-m-abadi** [reply]
Thanks, I already check them and fixed , but I think there is another issue that we don't know yet. 
Haven't you seen this issue before?

**Reply by PhiloNL** [reply]
I've seen the issue before, which was caused by an unclosed div.
